task2.py

```diff
@@ -0,0 +1,39 @@
+"""
+this progrom calculates maximum sum of hourglass in a 2D matrix
+"""
+def max_sum(array):
+    """
+    function to calculate maximum sum in the matrix
+    Parameters:
+       array (2D int) : recieves matrix from user
+    Returns:
+       max_sum (int) : value of max sum
+    """
+    row=6
+    column=6
+    max=-5000
+    for i in range(0,row-2):
+        for j in range(0,column-2):
+            sum= (array[i][j]+array[i][j+1]+array[i][j+2])+(array[i+1][j+1])+(array[i+2][j]+array[i+2][j+1]+array[i+2][j+2])
+            if sum>max:
+                max=sum
+            else:
+                continue
```

Comment:
Else is not needed here. 
We can also use max(max, sum) to get the maximum value